src/vuedraggable.d.ts

```diff
@@ -67,7 +67,17 @@ declare module 'vuedraggable' {
       clone: any;
       tag?: string | null;
       move: any;
-      componentData: any;
+      componentData: null | {
+        props?: {
+          [key: string]: any
+        }
+        attrs?: {
+          [key: string]: any
+        }
+        on?: {
+          [key: string]: any
```

Comment:
I would probably restrict the type here to functions only.
```suggestion
          [key: string]: Function
```

```ts
on: {
    // this allows for the following
    foo: () => {},
    // but disallows the following
    bar: ''
}
```